Gemfile

```diff
@@ -5,7 +5,7 @@ source 'https://rubygems.org'
 # Match ruby version in .ruby-version
 ruby File.read('.ruby-version')
 
-gem 'rails', '~> 6.1.0'
+gem 'rails', '~> 7.0.0'
```

Comment:
amazing!

Reply:
@cesy I think... I did a bunch of get rid of deprecations some time ago; which is why it's a bit cleaner.

Might go dependabot's version over this; but for buildpack-22..